src/ConfigManager.cpp

```diff
@@ -136,6 +136,48 @@ void ConfigManager::handleAPPost() {
     ESP.restart();
 }
 
+void ConfigManager::handleScanGet() {
+    DynamicJsonBuffer jsonBuffer;
+    JsonArray& jsonArray = jsonBuffer.createArray();
+
+    DebugPrintln("scanning WIFI networks...");
+    int n = WiFi.scanNetworks();
+    DebugPrintln("scan complete");
+    if (n == 0) {
+        DebugPrintln("no networks found");
+    } else {
+        DebugPrint(n);
+        DebugPrintln(" networks found:");
+
+        for (int i = 0; i < n; ++i) {
+            String ssid = WiFi.SSID(i);
+            int rssi = WiFi.RSSI(i);
+            String security = WiFi.encryptionType(i) == ENC_TYPE_NONE ? "none" : "enabled";
```

Comment:
Perhaps a bool here, to make it easier from a json perspective.

Reply:
added via JSON L168

```
{
"ssid": "ESP_21557E",
"strength": -80,
"security": false
},
{
"ssid": "NETGEAR24",
"strength": -71,
"security": true
},


00:40:25.783 -> Name: ESP_21557E - Strength: -80 - Security: none
00:40:25.783 -> Name: NETGEAR24 - Strength: -71 - Security: enabled

```